blockware/fireworks/src/main.cpp

```diff
@@ -213,16 +323,11 @@ void loop()
 
 void setup(void)
 {
-  Particles.reserve(23);
```

Comment:
is reserve better in initialize than in setup?

Reply:
It doesn't seem to make a big difference. I was playing around with it trying to figure out a memory issue and it seemed to succeed in reserving the space more often in initialize when there was a lot of memory pressure. Not sure that that means anything though.